pkgs/top-level/aliases.nix

```diff
@@ -421,6 +421,7 @@ mapAliases ({
   mathics = throw "mathics has been removed from nixpkgs, as it's unmaintained"; # added 2020-08-15
   matrique = spectral; # added 2020-01-27
   mbedtls_1_3 = throw "mbedtls_1_3 is end of life, see https://tls.mbed.org/kb/how-to/upgrade-2.0"; # added 2019-12-08
+  meme = throw "meme was renamed to meme-image-generator"; # added 2021-04-21
```

Comment:
```suggestion
  meme = meme-image-generator; # added 2021-04-21
```

Reply:
Ah, a gradual transition. Makes sense, and done.